frontend/src/components/IssueSidebar.vue

```diff
@@ -451,7 +450,7 @@ export default defineComponent({
       }
     };
 
-    const isDatePassed = (ts: number) => ts < now;
+    const isDayPassed = (ts: number) => ts < now - 24 * 60 * 60 * 1000;
```

Comment:
This looks not bad but a little bit weird
We used moment.js, maybe using [isSameOrAfter](https://momentjs.com/docs/#/query/is-same-or-after/) will be nice.

Reply:
Done. PTAL